src/main/java/com/manufacturing/dao/CarDao.java

```diff
@@ -4,15 +4,20 @@
 import java.util.List;
 import java.util.Optional;
 
-public interface CarDao {
+public interface CarDao extends GenericDao<Car, Long> {
+    @Override
     Car create(Car car);
 
+    @Override
     Optional<Car> get(Long carId);
 
+    @Override
     Car update(Car car);
 
+    @Override
     boolean delete(Long carId);
 
+    @Override
     List<Car> getAll();
```

Comment:
We don't need to add these methods here since they should be present in GenericDao (same for other interfaces)

Reply:
Check services as well, same mistake